### PR TITLE
javascript-forms - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-04-05 at 12 26 22 PM

### CODE REVIEW

**Comment by bbrown949** [comment]
forgot 1st screenshot

Screenshot 2023-04-05 at 1 33 15 PM

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  console.log('Value of:', event.target.name + ':', event.target.value);
+}
+
+const $userName = document.querySelector('#user-name');
+const $userEmail = document.querySelector('#user-email');
+const $userMessage = document.querySelector('#user-message');
+
+$userName.addEventListener('focus', handleFocus);
+$userEmail.addEventListener('blur', handleBlur);
+$userMessage.addEventListener('input', handleInput);
```
Comment (shawnkost) [inline javascript-forms/form-events.js]: You're missing 6 more event listeners. Each form input should have 3 event listeners, for each event.

**Review by shawnkost — CHANGES_REQUESTED** [review]
A change requested here Bryan!

File: javascript-forms/form-events.js (hunk 1 above)
Comment (bbrown949) [reply javascript-forms/form-events.js]: Updated

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**